importanize/formatters.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, unicode_literals
+
+import itertools
+import operator
+
+
+def get_normalized(i):
+    return map(operator.attrgetter('normalized'), i)
+
+
+class Formatter(object):
+    """Parent class for all formatters
+
+    Parameters
+    ----------
+    statement : ImportStatement instance
```

Comment:
nitpik - this is numpy doc - on first line you have var name and its type. on next then you add description. so `instance` is not necessary